client/dashboard/data/index.ts

```diff
@@ -59,15 +60,16 @@ export const fetchSites = async (): Promise< Site[] > => {
 			fields: SITE_FIELDS,
 		}
 	);
-	return sites;
+	return sites.map( ( site: Site ) => ( { ...site, slug: getSiteSlug( site ) } ) );
```

Comment:
If we are only going to use `getSiteSlug` in this file, could we move it here instead of the app-wide utils folder?

Reply:
Maybe we should just extract a small `mapRestAPISiteObject` function or something like that. `slug` might not be the only mapping we'll end-up doing.

Reply:
Good idea. I'll make a follow-up PR for this, thanks!